homeassistant/components/withings/__init__.py

```diff
@@ -116,37 +113,40 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.config_entries.async_update_entry(
             entry, data=new_data, options=new_options, unique_id=unique_id
         )
-    use_webhook = hass.data[DOMAIN][CONFIG][CONF_USE_WEBHOOK]
+    use_webhook = hass.data[DOMAIN][CONFIG].get(CONF_USE_WEBHOOK)
     if use_webhook is not None and use_webhook != entry.options[CONF_USE_WEBHOOK]:
```

Comment:
```suggestion
    use_webhook = hass.data[DOMAIN][CONFIG].get(CONF_USE_WEBHOOK)
    if (use_webhook := hass.data[DOMAIN][CONFIG].get(CONF_USE_WEBHOOK)) is not None and use_webhook != entry.options[CONF_USE_WEBHOOK]:
```
`use_webhook != entry.options[CONF_USE_WEBHOOK]` means YAML has priority but I guess that is what you want?

Reply:
For now yes, otherwise the setting of people aren't saved (and everyone would be reset to False unless they install a specific version)